SU-HSDO/suhumsci
# STN-507: Fix Spacing Regressions

# READY FOR REVIEW

## Summary
This PR fixes a few regressions that had been added in [STN-443](https://sparkbox.atlassian.net/browse/STN-443). See the testing instructions below for regressions. Regressions were caught with Backstop.js

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. - [x] Run `npm test`
1. - [x] Accordions should have reduced spacing (should match what is on staging)
    ![Screen Shot 2020-08-11 at 10 31 09 AM](https://user-images.githubusercontent.com/2486846/89910072-c6de9800-dbbd-11ea-8a1c-fae146873d36.png)
1. - [x] When an accordion is in a row well there should still be appropriate spacing around the accordion
    ![Screen Shot 2020-08-11 at 10 30 34 AM](https://user-images.githubusercontent.com/2486846/89910007-b2020480-dbbd-11ea-84f5-abd4574cfa94.png)
1. - [x] On the [Events](http://sparkbox-sandbox.suhumsci.loc/events) page edit the view and edit the format > settings. check the _Add views row classes_ box and confirm that adds spacing between the views.
    ![Default_Events__Content____QA_Sandbox__Stage_for_Colorful__Dev_for_Traditional_](https://user-images.githubusercontent.com/2486846/89909918-9696f980-dbbd-11ea-96d0-d82c78072d98.png)
1. - [x] [Carousels in the hero region](http://sparkbox-sandbox.suhumsci.loc/carousel-hero-image) should sit directly below the header/nav (when logged out)


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
